connectivity/manifests/client-egress-only-dns.yaml

```diff
@@ -15,16 +15,23 @@ spec:
         dns:
         - matchPattern: "*"
     toEndpoints:
-      - matchExpressions:
-          - { key: 'k8s-app', operator: In, values: [ "kube-dns", "coredns", "node-local-dns", "nodelocaldns" ] }
-          - { key: 'io.kubernetes.pod.namespace', operator: In, values: [ "kube-system" ] }
-  # When node-local-dns is deployed with local IP,
-  # Cilium labels its ip as world.
-  # This change prevents failing the connectivity
-  # test for such environments.
+    - matchLabels:
+        io.kubernetes.pod.namespace: kube-system
+        k8s-app: kube-dns
+    - matchLabels:
+        io.kubernetes.pod.namespace: kube-system
+        k8s-app: coredns
+    - matchLabels:
+        io.kubernetes.pod.namespace: kube-system
+        k8s-app: node-local-dns
```

Comment:
Why is this changed only in this policy but not the others? It also looks like `k8s-app: nodelocaldns` is missing.

Reply:
You should leave this part as it was.

Reply:
Done.

@tklauser @eminaktas please review again.